src/test/ObjectMap/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+# ceph_test_object_map
+add_executable(ceph_test_object_map
+  test_object_map.cc
+  KeyValueDBMemory.cc
+  )
+set_target_properties(ceph_test_object_map PROPERTIES COMPILE_FLAGS
+  ${UNITTEST_CXX_FLAGS})
+target_link_libraries(ceph_test_object_map
+  os
+  common
+  ${UNITTEST_LIBS}
+  global
+  ${EXTRALIBS}
+  ${CMAKE_DL_LIBS}
+  )
```

Comment:
can you use the `add_ceph_test()` for these targets? or are they not supposed to be part of `make check`?